### PR TITLE
Make it possible to disable the low battery warning

### PR DESCRIPTION
**Pull request by blackm0re**
Make it possible to disable the low battery warning in battery-widget and batteryarc-widget since many laptops have a dedicated LED for that purpose.

### CODE REVIEW

File: battery-widget/README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ It is possible to customize widget by providing a table with all or some of the
 | `warning_msg_text` | _Battery is dying_ | Text of the warning popup |
 | `warning_msg_position` | `bottom_right` | Position of the warning popup |
 | `warning_msg_icon` | ~/.config/awesome/awesome-wm-widgets/battery-widget/spaceman.jpg | Icon of the warning popup |
+| `disable_battery_warning` | `false` | Disable low battery warning |
```
Comment (streetturtle) [inline battery-widget/README.md:35]: Could you please invert this option. I know it sounds like not a big deal, but I really believe that `enable_battery_warning` would be better. Also the double negation here ("disable" together with "false") might sound a bit misleading. 
I would really appreciate it.

Comment (blackm0re) [reply battery-widget/README.md:35]: enable_battery_warning and the default being... true?

Comment (streetturtle) [reply battery-widget/README.md:35]: Yep

Comment (blackm0re) [reply battery-widget/README.md]: Updated the PR

**Pull request merged**